### DISCUSSION TITLE
About Coordinate Saving

### DISCUSSION THREAD

**Discussion by Boldguyx** [mattieserver/netbox-topology-views#400] [General]
Hey guys,

I have a question:
when I create a coordinate group, add a couple of devices to that group and give them coordinates.

after applying the group filter if I move a couple of devices, should the new position be saved if the following configuration is set?

PLUGINS_CONFIG = {
    'netbox_topology_views': {
        'static_image_directory': 'netbox_topology_views/img',
        'allow_coordinates_saving': True,
        'always_save_coordinates': True,
        'enable_circuit_terminations': True
    }
}

**Comment by dreng** [comment]
First of all, you don't need to add coordinates manually. You can, of course, but coordinates are being saved automatically if you drag devices to a new position.

To answer your question, yes, the new position should be saved. You should see a greyed out checkbox "Save Coordinates" in the filter section because of the "always_save_coordinates" option. Make sure you've set the correct coordinates group in the filter section.

Are you able to check the webserver log? You should see "PATCH" entries.

**Reply by Boldguyx** [reply]
Apparently, I did something that fixed the issue, but I am not sure what.

because after restarting the server coordinate saving is working indeed.

I really appreciate your assistance.

 thanks.
